examples/sgemm/SGEMM_main.cpp

```diff
@@ -231,7 +232,18 @@ int main(int argc, char **argv) {
     } else {
         printf("%s\n", argv[0]);
         printf("\nInvalid number of inputs\n");
-        getchar();
+        exit(-1);
+    }
+
+    int programCount = SGEMM_get_program_count();
+    int tileSize = SGEMM_get_tile_size();
+    if (K % programCount != 0 || K % tileSize != 0) {
+        printf("\nNumber of columns in Matrix B must be a multiple of %d and %d!\n", programCount, tileSize);
```

Comment:
Let's have additional clarification what these numbers are. i.e.
``"...of %d (target width) and %d (tile size)!\n"``

Reply:
Done.